.github/workflows/goreleaser.yml

```diff
@@ -24,10 +24,10 @@ jobs:
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
         with:
           fetch-depth: 0
-      - name: Run Tests
-        uses: ./.github/actions/test.yaml
-      - name: Run Lints
-        uses: ./.github/actions/lint.yaml
+      # - name: Run Tests
```

Comment:
Can we bring these back? This is an important part of the release process.

Reply:
the reason why I did this is that workflow didn't worked as it didn't find these action files under `.github/actions`.

Reply:
Ah my bad, fixing this in #311.

Reply:
@another-rex I think the problem still continues. I also do not think that this is the way of referencing different workflows within the workflow.

<img width="1124" alt="Screen Shot 2023-03-25 at 3 29 07 PM" src="https://user-images.githubusercontent.com/16693043/227717341-d5927b11-62c7-4a8b-81c7-118c3494dddd.png">

Reply:
You're right, I misunderstood how the uses line works. I updated and tested it in #315, which seems to work in my fork.